components/atoms/Accordion.js

```diff
@@ -13,7 +13,7 @@ export function Accordion(props) {
   const arrowColor = "#D3080C";
 
   return (
-    <div id="accordion" className="flex mb-2 container">
+    <div id="accordion" className="flex mb-2">
```

Comment:
id needs to be unique. The way it is now there are 7 Accordions with the same id.